### ISSUE TITLE
Do a quick temporary offline validation to prevent `No Data Available`

### ISSUE THREAD

**Issue by ydahhrk** [NICMx/FORT-validator#70]
Just a wish. Probably low priority.

Experience has shown that clients often don't clearly report [No Data Available (yet)](https://datatracker.ietf.org/doc/html/rfc8210#section-12), so this tends to confuse users:

	First validation cycle has begun, wait until the next notification to connect your router(s)
	...
	...
	...
	First validation cycle successfully ended, now you can connect your router(s)

Proposed solution: If there is existing cache, do an offline quick in-place validation to produce existing (but admittedly outdated) state, before starting the RTR server.

Alternatively or in case there's no cache: Produce an empty data set (assuming it's possible).
